test/dlc_tests/conftest.py

```diff
@@ -92,20 +92,45 @@ def ec2_instance(
 ):
     print(f"Creating instance: CI-CD {ec2_key_name}")
     key_filename = test_utils.generate_ssh_keypair(ec2_client, ec2_key_name)
-    instances = ec2_resource.create_instances(
-        KeyName=ec2_key_name,
-        ImageId=ec2_instance_ami,
-        InstanceType=ec2_instance_type,
-        IamInstanceProfile={"Name": ec2_instance_role_name},
-        TagSpecifications=[
-            {
-                "ResourceType": "instance",
-                "Tags": [{"Key": "Name", "Value": f"CI-CD {ec2_key_name}"}]
-            },
-        ],
-        MaxCount=1,
-        MinCount=1,
-    )
+    extra_volume_size_mapping = [
+                        {
+                            'DeviceName': '/dev/sda1',
+                            'Ebs': {
+                                'VolumeSize': 300,
+                            },
+                        },
+                    ]
+    if ("performance" in ec2_key_name) and ("mxnet" in ec2_key_name) and ("training_cpu" not in ec2_key_name):
+        instances = ec2_resource.create_instances(
+            BlockDeviceMappings=extra_volume_size_mapping,
+            KeyName=ec2_key_name,
+            ImageId=ec2_instance_ami,
+            InstanceType=ec2_instance_type,
+            IamInstanceProfile={"Name": ec2_instance_role_name},
+            TagSpecifications=[
+                {
+                    "ResourceType": "instance",
+                    "Tags": [{"Key": "Name", "Value": f"CI-CD {ec2_key_name}"}]
+                },
+            ],
+            MaxCount=1,
+            MinCount=1,
+        )
+    else:
+        instances = ec2_resource.create_instances(
+            KeyName=ec2_key_name,
+            ImageId=ec2_instance_ami,
+            InstanceType=ec2_instance_type,
+            IamInstanceProfile={"Name": ec2_instance_role_name},
+            TagSpecifications=[
+                {
+                    "ResourceType": "instance",
+                    "Tags": [{"Key": "Name", "Value": f"CI-CD {ec2_key_name}"}]
+                },
+            ],
+            MaxCount=1,
+            MinCount=1,
+        )
```

Comment:
nit: You can clean this up by creating a dictionary containing each of these parameter assignments, and then passing that dictionary directly to the `create_instances` function.

E.g.:-
```
params = {
    "KeyName": ec2_key_name,
    "ImageId": ec2_instance_ami,
    ...
}
if "performance" in ec2_key_name and "mxnet" in ec2_key_name and "training_cpu" not in ec2_key_name:
    params["BlockDeviceMappings"] = extra_volume_size_mapping
instances = ec2_resource.create_instances(**params)
```

Reply:
Will polish this part. Thanks for coding suggestion.